data_rentgen/server/schemas/v1/lineage.py

```diff
@@ -116,12 +106,39 @@ class RunLineageQueryV1(BaseLineageQueryV1):
 
 
 class LineageRelationV1(BaseModel):
```

Comment:
Actually, this class could be removed. Just copy `to` and `from_` fields to inherited models